src/applications/claims-status/tests/components/claim-status-tab/RecentActivity.unit.spec.jsx

```diff
@@ -981,13 +985,15 @@ describe('<RecentActivity>', () => {
                 'Your claim moved into Step 3: Evidence gathering, review, and decision',
               );
               getByText('Request for you');
-              getByText('We opened a request for "5103 Evidence Notice"');
+              getByText(
+                'We opened a request for "List of evidence we may need (5103 notice)"',
+              );
               expect($('va-pagination', container)).not.to.exist;
             });
           },
         );
         context(
-          'when cst5103UpdateEnabled and has an Automated 5103 Notice Response item',
+          'when cst5103UpdateEnabled disabled and has an Automated 5103 Notice Response item',
```

Comment:
In the below test we're setting `store` with both feature flags disabled, so I updated the context name here to reflect that.